### DISCUSSION TITLE
I'm probably missing some detail

### DISCUSSION THREAD

**Discussion by MrAmericanMike** [eslint/eslint#16234] [Q&A]
I'm having an issue with eslint complaining on css files.

In an empty folder I start an NPM project, then do `npx eslint --init` and after answering some questions I end with this configuration file.

```json
{
	"env": {
		"browser": true,
		"es2021": true
	},
	"extends": "eslint:recommended",
	"overrides": [],
	"parserOptions": {
		"ecmaVersion": "latest",
		"sourceType": "module"
	},
	"rules": {
		"indent": [
			"error",
			"tab"
		],
		"linebreak-style": [
			"error",
			"windows"
		],
		"quotes": [
			"error",
			"double"
		],
		"semi": [
			"error",
			"always"
		]
	}
}
```

Then I create for example a src folder and  public one with a styles.css file on it with this content.

```css
* {
	margin: 0px;
	padding: 0px;
	box-sizing: border-box;
}
```

And the * gets underlined in red in my VSCode and the error says:
```
<element>
Selector Specificity: (0, 0, 0)

Parsing error: Unexpected token *eslint
```

I believe it's probably some missing plugin, but I'm not positive about this. (I have some Svelte projects where having svelte3 plugin makes this error go away, but at the same time some SvelteKit projects the error comes back)

What could be the cause and what would be the definitive fix?

Thanks in advance.

**Comment by nzakas** [comment]
This doesn't appear related to ESLint and is more likely related to a plugin you have in VS Code. That specificity is not something ESLint would report.

Try running ESLint on the command line to see if you get the same result:

```
npx eslint src/
```

**Comment by MrAmericanMike** [comment]
Digging more into the settings, I found out that this is the issue.

Having on my workspace settings:

```json
"settings": {
		"eslint.validate": [
			"css"
		]
	}
```

Cause the issue

PS: The output of the command was:
```
Oops! Something went wrong! :(

ESLint: 8.23.0

No files matching the pattern "src/" were found.
Please check for typing mistakes in the pattern.
```
